src/dotnet/Audio.UI.Blazor/Components/AudioRecorder/AudioRecorder.cs

```diff
@@ -11,15 +11,15 @@ public record AudioRecorderState(ChatId ChatId) : IHasId<Ulid>
 
 public class AudioRecorder : IAudioRecorderBackend, IAsyncDisposable
 {
-    private readonly ILogger<AudioRecorder> _log;
-    private ILogger? DebugLog => DebugMode ? _log : null;
-    private bool DebugMode => Constants.DebugMode.AudioRecording;
-
-    private readonly Session _session;
-    private readonly IJSRuntime _js;
     private readonly IMessageProcessor<IAudioRecorderCommand> _messageProcessor;
-    private IJSObjectReference? JSRef { get; set; }
-    private DotNetObjectReference<IAudioRecorderBackend>? BlazorRef { get; set; }
+    private IJSObjectReference? _jsRef;
+    private DotNetObjectReference<IAudioRecorderBackend>? _blazorRef;
+
+    private ILogger<AudioRecorder> Log { get; }
+    private ILogger? DebugLog => DebugMode ? Log : null;
```

Comment:
it does not look convenient

Reply:
Could you please explain, what`s wrong there